src/browser.js

```diff
@@ -79,6 +81,12 @@ window.onload = function() {
     navigateTo(CLEVER_LOGIN_URL);
   };
 
+  document.querySelector('#google-login').onclick = function() {
+    // Opens link in default browser
+    require('electron').shell.openExternal(SIGN_IN_URL + '?user_return_to=/maker/display_google_oauth_code&maker=true');
```

Comment:
😮 Does this work?

Back when I implemented [walled garden navigation](https://github.com/code-dot-org/browser/pull/9) (has since been removed) I remember having to worry about the difference between the main Electron process and the renderer process when figuring out how to open a link in the default browser.  There are notes in that PR about how this had to happen in the main process, but you have it happening in the renderer process.

I assume you tested this locally, so I'm not sure why it's not an issue anymore... would love to understand that better.  Or maybe I significantly overengineered that earlier feature!

Reply:
:woman_shrugging: It works for me (Linux and Chrome), but this is a good reminder that I should check with other folks to make sure it works on their set-up.